### PR TITLE
Port the platform service to be globally installed.

### PR DESCRIPTION
**Pull request by cramforce**
Avoids having its code in extensions.

### CODE REVIEW

**Comment by cramforce** [comment]
Not ready for review.

**Comment by cramforce** [comment]
About a 1.5 KB win in the extensions that use it.

File: test/functional/test-platform.js [hunk 1]
```diff
@@ -5,7 +5,7 @@
  * you may not use this file except in compliance with the License.
  * You may obtain a copy of the License at
  *
- *      http://www.apache.org/licenses/LICENSE-2.0
+ *      http://www.apache.org/licenses/LICENSE-2.0fix
```
Comment (erwinmombay) [inline test/functional/test-platform.js]: intentional?

Comment (cramforce) [inline test/functional/test-platform.js]: Lol, no :)

**Comment by erwinmombay** [comment]
LGTM

**Pull request merged**